tools/nl/embeddings/utils.py

```diff
@@ -163,22 +166,6 @@ def dedup_texts(df: pd.DataFrame) -> Tuple[Dict[str, str], List[List[str]]]:
   return (text2sv_dict, dup_sv_rows)
 
 
-def _download_file_from_gcs(ctx: Context, file_name: str) -> str:
-  """Downloads the specified file_name from GCS to the ctx.tmp folder.
-
-  Args:
-    ctx: Context which has the GCS bucket information.
-    file_name: the GCS bucket name for the file.
-
-  Returns the path to the local directory where the file was downloaded to.
-  ```
-  """
-  local_file_path = os.path.join(ctx.tmp, file_name)
-  blob = ctx.bucket.get_blob(file_name)
-  blob.download_to_filename(local_file_path)
-  return local_file_path
-
-
 def _download_model_from_gcs(ctx: Context, model_folder_name: str) -> str:
```

Comment:
TODO:  Can this use download_folder?  (We can move it to shared)

Reply:
Yes. Elaborated the TODO below.